generated/nifgen/nifgen/session.py

```diff
@@ -3918,7 +3918,8 @@ def import_attribute_configuration_buffer(self, configuration):
         '''
         vi_ctype = _visatype.ViSession(self._vi)  # case S110
         size_in_bytes_ctype = _visatype.ViInt32(0 if configuration is None else len(configuration))  # case S160
-        configuration_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_to_bytes(configuration), library_type=_visatype.ViInt8)  # case B520
+        converted_configuration = _converters.convert_to_bytes(configuration)  # case B520
+        configuration_ctype = get_ctypes_pointer_for_buffer(value=converted_configuration, library_type=_visatype.ViInt8)  # case B520
         error_code = self._library.niFgen_ImportAttributeConfigurationBuffer(vi_ctype, size_in_bytes_ctype, configuration_ctype)
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
```

Comment:
I had expected this to affect NI-FGEN's multiple "write waveform" and "create waveform" methods. Do you know why it didn't? Could you double check?

Reply:
This change affects only calls to `get_ctypes_pointer_for_buffer()` where a converter is used on the `value` beforehand.

NI-FGEN's write/create waveform methods don't use the converters, they call `get_ctypes_and_array()`:
https://github.com/ni/nimi-python/blob/5740196f069e45fd4ac3d6619fe2bebc67d73535/generated/nifgen/nifgen/session.py#L1481
https://github.com/ni/nimi-python/blob/5740196f069e45fd4ac3d6619fe2bebc67d73535/generated/nifgen/nifgen/session.py#L2817